uiauto/lib/element-patch/helper-patch.js

```diff
@@ -149,4 +149,15 @@
     return this.value() === 1;
   };
 
+  UIAElement.prototype.childElementsFrames = function () {
+    var childElements = this.elements();
+    var frames = [];
+    childElements.toArray().forEach(function(el) {
```

Comment:
I assume `forEach` has been replaces with `map` since ES6

Reply:
so the result might be presented as 

```js
return   childElements
  .toArray()
  .map(x => x.rect());
```

Reply:
unfortunately the uiauto code runs in an ES4 (i think) context.